src/App.js

```diff
@@ -18,12 +26,74 @@ app.get("/users", async function(req, res) {
 });
 
 app.get("/put/:name", function(req, res) {
-  var user = new User({ username: req.params.name, passord:"demo" });
+  var user = new User({ username: req.params.name, password:"demo" });
   user.save();
   console.log("Added User " + req.params.name);
   res.send("Added User " + req.params.name);
 });
 
+app.post("/post/google", async function(req, res) {
+  if (!req.body) return res.sendStatus(400)
+
+  const tokenInfoRes = await fetch(`https://www.googleapis.com/oauth2/v3/tokeninfo?id_token=${req.body.tokenId}`);
+  const payload = await tokenInfoRes.json();
+
+  addUser = true
+  const allUsers = await User.find()
```

Comment:
you can edit the await User.find() to be await User.find({email: payload.email, googleAuth: true}) to get the correct user, and then just not do anything if you dont have a user